tools/test.py

```diff
@@ -34,25 +19,14 @@ def main():
 
     enable_ansi_colors()
 
-    test_cases = [
-        TestSetup,
-        TestUtil,
-        TestTarget,
-        JsUnitTests,
-        TestFetch,
-        TestRepl,
-        TestDenoDir,
-        TestBenchmark,
-        TestIsTty,
-    ]
-    test_cases += permission_prompt_tests()
-    test_cases += complex_permissions_tests()
-    # It is very slow, so do TestFmt at the end.
-    test_cases += [TestFmt]
-
-    with http_server.spawn():
-        run_tests(test_cases)
```

Comment:
This removes the pattern matching functionality of test.py, which can be useful...

I couldn't see any matching functionality in `cargo test` ootb .

Reply:
I think the first arg of cargo test is a pattern. For example, I guess in this branch "cargo test benchmark"  would run the benchmark test

Reply:
Previously the -p pattern matched the method as well as the class (file).

Reply:
I see, tools/test.py is still the test runner, but only superficially. It should be relatively easy to pass the `-p` parameter through to `cargo test`.